### PR TITLE
[User profile] Avatar image broken when user has no profile pic

### LINKED ISSUE #236: [User profile] Avatar image broken when user has no profile pic
We may need to implement the circle w/the first letter of the user name as is done in other profile/avatar components.

### PR DESCRIPTION
**Pull request by parulraheja98**
image

image

Closes #236

### CODE REVIEW

File: web-ng/src/app/components/project-header/project-header.component.html [hunk 1]
```diff
@@ -30,12 +30,18 @@
...
+        *ngIf="user.photoURL; else defaultPhoto"
         src="{{ user.photoURL }}=s64"
-        alt="{{ user.displayName }}"
         class="user-thumb"
-        style="cursor: pointer;"
         (click)="openProfileDialog($event)"
       />
+      <ng-template #defaultPhoto>
+        <span
+          [attr.data-letters]="user.displayName.charAt(0)"
```
Comment (gino-m) [inline web-ng/src/app/components/project-header/project-header.component.html:40]: How does this actually work? What component causes the letter to be read from the data-letters and attribute? And why do we need :before pseudo-elements in CSS?

File: web-ng/src/app/components/user-profile-popup/user-profile-popup.component.scss [hunk 2]
```diff
@@ -42,10 +45,24 @@
...
-  margin-top: 10px ;
+  margin-top: 10px;
   text-align: center;
 }
 
 .btn-signout {
   margin-top: 20px;
 }
+
+[data-letters]:before {
```
Comment (gino-m) [inline web-ng/src/app/components/user-profile-popup/user-profile-popup.component.scss:56]: Duplicated UI widgets should implemented as a separate component (e.g., user-avatar-component).

Comment (parulraheja98) [reply web-ng/src/app/components/user-profile-popup/user-profile-popup.component.scss]: Done..

File: web-ng/src/app/components/project-header/project-header.component.html (hunk 1 above)
Comment (parulraheja98) [reply web-ng/src/app/components/project-header/project-header.component.html]: Remove pseudo attr.

**Pull request merged**